lib/wanda/execution/evaluation.ex

```diff
@@ -224,4 +237,51 @@ defmodule Wanda.Execution.Evaluation do
   defp result_weight(:critical), do: 2
   defp result_weight(:warning), do: 1
   defp result_weight(:passing), do: 0
+
+  defp compute_values(values, facts, env) do
+    Enum.map(values, &value_from_conditions(&1, facts, env))
+  end
+
+  defp value_from_conditions(
```

Comment:
Let's put this function just after the `has_some_fact_gathering_error` function. Usually, we put the elixir function as they appear and are used in the code

Reply:
`eval_values` as xabi proposed